src/Widgets/TableWidget/TableWidget.scss

```diff
@@ -0,0 +1,184 @@
+@import "src/assets/stylesheets/variables";
+
+.bg-dark-image,
+.bg-brand-primary,
+.bg-brand-secondary,
+.bg-grey,
+.bg-greydark,
+.bg-greymiddle,
+.bg-greymiddle {
+  footer & {
```

Comment:
If I'm not mistaken, the `footer` part can be removed, since the general `.table-widget` overwrite is strong enough anyway.